sdk/src/Handlers/EntityFramework/AWSXRayRecorder.Handlers.EntityFramework.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Company>Amazon.com, Inc</Company>
+    <Product>Amazon Web Service X-Ray Recorder</Product>
+    <Copyright>Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.</Copyright>
+    <AssemblyName>AWSXRayRecorder.Handlers.EntityFramework</AssemblyName>
+    <RootNamespace>Amazon.XRay.Recorder.Handlers.EntityFramework</RootNamespace>
+    <Authors>Amazon Web Services</Authors>
+    <Description>This package contains libraries to trace SQL queries through Entity Framework Core.</Description>
+    <PackageLicenseUrl>http://aws.amazon.com/apache2.0/</PackageLicenseUrl>
+    <PackageProjectUrl>https://aws.amazon.com/documentation/xray/</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/aws/aws-xray-sdk-dotnet</RepositoryUrl>
+    <PackageIconUrl>https://sdk-for-net.amazonwebservices.com/images/AWSLogo128x128.png</PackageIconUrl>
+    <PackageTags>AWS Amazon cloud AWSXRay XRay</PackageTags>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../../../../buildtools/local-development.snk</AssemblyOriginatorKeyFile>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;1591;</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="2.0.0" />
```

Comment:
I don't think you need the Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore dependency.